_docs/tasks/ingress.md

```diff
@@ -134,10 +151,55 @@ to configure ingress behavior.
    
    _Remark:_ Envoy currently only allows a single TLS secret in the ingress since SNI is not yet supported.
    
+1. Determine the secure ingress URL:
+ 
+    If your cluster is running in an environment that supports external load balancers,
+    use the ingress' external address:
+ 
+    ```bash
+    kubectl get ingress secured-ingress -o wide
+    ```
+    
+    ```
+    NAME              HOSTS     ADDRESS                 PORTS     AGE
+    secured-ingress   *         130.211.10.121          80, 443   1d
+    ```
+ 
+    ```bash
+    export SECURE_INGRESS_URL=130.211.10.121
+    ```
+ 
+    If load balancers are not supported, use the ingress controller's hostIP:
+    
+    ```bash
+    kubectl get po -l istio=ingress -o jsonpath='{.items[0].status.hostIP}'
+    ```
+ 
+    ```
+    169.47.243.100
+    ```
+ 
+    along with the istio-ingress service's nodePort for port 443:
+    
+    ```bash
+    kubectl get svc istio-ingress
+    ```
+    
+    ```
+    NAME            CLUSTER-IP     EXTERNAL-IP   PORT(S)                      AGE
+    istio-ingress   10.10.10.155   <pending>     80:31486/TCP,443:32254/TCP   32m
+ 
+    ```
+    
+    ```bash
+    export SECURE_INGRESS_URL=169.47.243.100:32254
+ 
+    ```
+    
 1. Access the secured httpbin service using _curl_:
 
    ```bash
-   curl -k https://$INGRESS_URL/html
+   curl -k https://$SECURE_INGRESS_URL/html
```

Comment:
Yes, I know it turns out to be the same in the "load balancer" case, but in the nodePort case, it's different (<ip>:<nodeport for port 80> vs <ip>:<nodeport for port 443>). Wasn't sure how best to unify the instructions for the two cases, so this is what I came up with. I'll merge this for now, and see if we can come up with a better way to document it later.